scenarioo-client/app/components/detailarea/detailarea.component.ts

```diff
@@ -53,6 +53,20 @@ export class DetailareaComponent {
     @Output('valueChange')
     panelCollapsed: EventEmitter<boolean> = new EventEmitter<boolean>();
 
+    ngOnInit(): void {
+        this.setHeightOfDetailarea();
+    }
+
+    @HostListener('window:resize', ['$event'])
+    onResize() {
+        this.setHeightOfDetailarea();
+    }
+
+    setHeightOfDetailarea() {
+        const headerHeight = document.getElementById('sc-metadata-panel').offsetTop + 2;
+        document.getElementById('sc-metadata-panel').style.height = 'calc(100vh - ' + headerHeight + 'px)';
```

Comment:
this should be set through a binding not directly in the DOM. E.g. create a member variable metaDataPanelHeight and reference it in the the template. I'm not sure though, if it's possible since the calculation is rather complicated.

Reply:
To access DOM elements direclty in Angular use:
```import {ElementRef,Renderer2} from '@angular/core';
@ViewChild('someVar') el:ElementRef;

constructor(private rd: Renderer2) {}

ngAfterViewInit() {
      console.log(this.rd); 
      this.el.nativeElement.focus();      //<<<=====same as oldest way
}```

https://stackoverflow.com/questions/38944725/how-to-get-dom-element-in-angular-2/39679241

Reply:
could the same not simply be achieved by setting minHeight to 100% for the details area?

Reply:
> could the same not simply be achieved by setting minHeight to 100% for the details area?

This is unfortunately not possible. By setting minHeight to 100%, it doesn't subtract the space above the detail area.